my-app/src/home-task-1/create-pure-react-component.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export class CreateReactPureComponent extends React.Component {
```

Comment:
To create PureComponent you should extends from the corresponding class: React.PureComponent